src/views/network/view_network.cpp

```diff
@@ -81,22 +81,42 @@ namespace big
 
 	void render_session_switcher()
 	{
+		if (g_pointers->m_gta.m_region_code == nullptr)
+			return;
+
+		static int selected_region_index     = -1;
+		static bool region_updated = false;
+
+        std::string region_str = (selected_region_index == -1) ? "SESSION_SELECT_COMBO"_T.data() : regions[*g_pointers->m_gta.m_region_code].name;
```

Comment:
Same here, sorry for not catching this earlier however using clang-format would've fixed this.

You can get it by installing the LLVM package on Windows (you need to add it to your path for it to start working).
![clang-format-llvm](https://github.com/YimMenu/YimMenu/assets/24669514/295ae2f6-4ae2-4e31-8286-fd3a822ede1e)

Reply:
I missed this one as well, which actually brings me to a general question about code styling moving forward.

Would you prefer to see clang formatting applied against entire source files for consistency, or only to new code? If I were to execute clang formatting against the entire source file, we'd see a LOT more changes which would be functionally unnecessary, but would make the old code conform to the style.

Reply:
@rkwapisz, I have clang-format tied to my document saving so it will affect the whole file whether I like it or not.

In general it's better as old code will also conform to the styling guidelines, however gir489 might see some of these as unnecessary.